pkg/broker/ipsec_psk_test.go

```diff
@@ -0,0 +1,29 @@
+package broker
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const testPSKLen = 32
+
+var _ = Describe("ipsec_psk handling", func() {
```

Comment:
Ah, these tests are great, thanks! I was wondering about how best to do add tests, thanks for the informative example. :)